src/frontend/App/utils/utils.ts

```diff
@@ -114,3 +115,25 @@ export const groupBy = <T, K extends keyof any>(list: T[], getKey: (item: T) =>
         previous[group].push(currentItem);
         return previous;
     }, {} as Record<K, T[]>);
+
+export const fødselsdatoTilAlder = (fødselsdato: string): number => {
```

Comment:
Det finnes nå en annen funksjon for denne `datoTilAlder`  (nylig lagt til)

Reply:
Nice, byttet i 9adb2f7